test/.eslintrc.js

```diff
@@ -11,14 +11,17 @@ module.exports = {
         'afterAll': true,
         'test': true,
         'jest': true,
-        'page': true
+        'page': true,
+        'browserlist': true
     },
 
 
     'rules': {
         'compat/compat': 'off',
         'max-lines': 'off',
         'no-restricted-globals': 'off',
-        'promise/no-native': 'off'
+        'promise/no-native': 'off',
+        'key-spacing': 'off',
+        'import/no-commonjs': 'off'
```

Comment:
Please do not disable this rule. This one is essential for tree-shaking in the SDK. If there are specific cases where you need to disable it, do it line by line.